library/cwm/src/lib/cwm/table.rb

```diff
@@ -33,17 +60,19 @@ class Table < CustomWidget
     #   def items
     #     [
     #       [:first_user, "Joe", "Doe"],
-    #       [:best_user, "Chuck", "Norris"]
+    #       TableItem.new(:best_user, ["Chuck", "Norris"])
```

Comment:
I am fine even if you declare plain array deprecated and use just that new table item. Just can you add helper for creating item? like `item(:best_user, ["Chuck", "Norris"])` ? or table_item if there are collision with UIShortcuts.

Reply:
Actually, you don't even need to wrap it into that `Cell()` if it's jut a plain string. We can save that intermediate step if it's just an array of strings.

See also
  https://github.com/yast/yast-ycp-ui-bindings/blob/master/examples/Table-nested-items.rb#L48

Only if a cell needs an icon or a sort key it is necessary to use `Cell()` around the content.

Reply:
Strictly speaking, `item` would not collide with the UIShortcuts because the shortcut is `Item` (with capital "I"). But maybe they are still too similar

Reply:
Well, I see exactly the same happens with `cell` and `Cell`, so I guess it's fine.

Reply:
Added the `item` helper.